openreview/conference/templates/reviewerWebfield.js

```diff
@@ -183,10 +183,17 @@ var displayHeader = function(headerP) {
 
   var reducedLoadP = $.Deferred().resolve(0);
   if (REVIEW_LOAD > 0) {
+    userIds = _.union([user.profile.id], user.profile.emails);
     reducedLoadP = Webfield.get('/notes', { invitation: CONFERENCE_ID + '/-/Reduced_Load'})
     .then(function(result) {
       if (result.notes && result.notes.length) {
-        return result.notes[0].content.reviewer_load;
+        if (result.notes.length == 1) {
+          return result.notes[0].content.reviewer_load;
+        } else {
+          //If there is more than one there might be a Program Chair
+          loads = _.filter(result.notes, function(note) { return userIds.indexOf(note.content.user) >= 0;});
```

Comment:
Missing `var` here. 

I think in this case we should just use the native filter, `result.notes.filter(...`